pkg/apis/admission/v1alpha1/types.go

```diff
@@ -0,0 +1,178 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	CanonicalName = "AdmissionControlConfiguration"
+)
+
+// +genclient=true
+
+// AdmissionControlConfiguration is the cluster-wide configuration of default
+// initializers and external admission webhooks that are applied to each
+// resource.
+type AdmissionControlConfiguration struct {
+	metav1.TypeMeta `json:",inline"`
+
+	// ResourceInitializers is a list of resources and their default initializers
```

Comment:
It looks like the convention in k8s is omit the subject in the sentence. I'll revise later.